v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerShardedMigrationIT/spanner-schema.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS Users (
+    id INT64 NOT NULL,
+    name STRING(200),
+    age_spanner INT64,
+    migration_shard_id STRING(50),
+) PRIMARY KEY (migration_shard_id, id);
+
+CREATE TABLE IF NOT EXISTS Movie (
+    id1 INT64 NOT NULL,
+    id2 INT64 NOT NULL,
+    name STRING(200),
+    actor INT64,
+    migration_shard_id STRING(50),
+) PRIMARY KEY (id2, id1, migration_shard_id);
```

Comment:
Can you point me to the test case where data types are covered?
I agree that the data type coverage can be in a non-sharded test

Reply:
The test case is being added in this PR https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1323